docs/components/skeleton.md

```diff
@@ -165,12 +167,137 @@ to perform any expensive querying on first load.
 </template>
 ```
 
+## KSkeletonBox
+
+KSkeleton package uses a component to render the placeholder content 
+`<KSkeletonBox>`. It can be used as a component primitive to create your own
+custom placeholder components.
+
+<KSkeletonBox />
+<KSkeletonBox width="2" height="2"/>
+<KSkeletonBox width="5" height="2"/>
+<KSkeletonBox width="50" height="1"/>
+<KSkeletonBox width="100" height="2"/>
```

Comment:
you should probably mention here the accept values for width and height since u have a validator there for certain values